File: proxies/java/src/main/java/com/dvc_harness/proxy/data/DataStore.java
package com.dvc_harness.proxy.data;

import java.util.Hashtable;

import com.devcycle.sdk.server.cloud.api.DevCycleCloudClient;
import com.devcycle.sdk.server.local.api.DevCycleLocalClient;
import com.devcycle.sdk.server.common.model.*;

public class DataStore {
    public static Hashtable<String, DevCycleLocalClient> LocalClients = new Hashtable();
    public static Hashtable<String, DevCycleCloudClient> CloudClients = new Hashtable();

    public static Hashtable<String, Hashtable> CommandResults = new Hashtable();
}

File: proxies/dotnet/DataStore.cs
using DevCycle.SDK.Server.Cloud.Api;
using DevCycle.SDK.Server.Local.Api;
using dotnet.Models;
using OpenFeature;

public class DataStoreClient
{
    public object? DvcClient { get; set; }
    public IFeatureClient? OpenFeatureClient { get; set; }
    public bool IsOpenFeature { get; set; }
}

class DataStore
{
    public static Dictionary<string, DevCycleLocalClient> LocalClients;
    public static Dictionary<string, DevCycleCloudClient> CloudClients;
    public static Dictionary<string, OpenFeatureClientAdapter> OpenFeatureClients;

    public static Dictionary<string, object> Commands;

    static DataStore()
    {
        CloudClients = new Dictionary<string, DevCycleCloudClient>();
        LocalClients = new Dictionary<string, DevCycleLocalClient>();
        OpenFeatureClients = new Dictionary<string, OpenFeatureClientAdapter>();
        Commands = new Dictionary<string, object>();
    }
}
